app/src/main/java/com/example/scoutconcordia/ShuttleInfo.java

```diff
@@ -0,0 +1,587 @@
+package com.example.scoutconcordia;
+
+import java.util.Calendar;
+
+
+// To use this class, the app has to essentially create an object of 'ShuttleInfo'.
+// Then, the app needs to call either of two methods, depending on the location of the user.
+// If the user is at SGW, the object should call 'getNextEarliestTimeFromSGW()'.
+// If the user is at Loyola, the object should call 'getNextEarliestTimeFromLoyola()'.
+
+
+public class ShuttleInfo {
+
+    // These attributes hold the shuttle times form specific campuses on given days
+    String[] schedule_monToThurs_Loyola = {"07:30",
+            "07:40",
+            "07:55",
+            "08:20",
+            "08:35",
+            "08:55",
+            "09:10",
+            "09:30",
+            "09:45",
+            "10:20",
+            "10:35",
+            "10:55",
+            "11:10",
+            "11:30",
+            "12:00", "12:30", "13:00", "13:30", "14:00", "14:30", "15:00", "15:30", "16:00", "16:30", "17:00", "17:30", "18:00", "18:30", "19:00", "19:30",
```

Comment:
Pertains to coding conventions/uniformity:
All other times are listed individually on a line, however times on this line are not. 
We should make it uniform. Either have several departures that are in the same hour be on a single line, or have every departure time be on an individual line. 

I would prefer having departures during the same hour on a single line. 
@jaychenblue, what do you think?

Reply:
I agree. All departures for the same hour could be on the same line. Either that or every departure should be on its own line. There should be a consistent format that's followed.

I'm also not sure having attributes being arrays is a good idea, especially if these times are supposed to be hard coded. @JenYn, thoughts?

Reply:
Would a list be more suitable? @jaychenblue

Reply:
@JenYn @jaychenblue I don't see the functional advantage of a list over an array in this case, specifically because these times are set in stone and hard-coded.

Reply:
@vicmanea 

One possibility is using an enum type.

Another possibility is to change the arrays to private and create getters for the array.

The issue isn't functionality or performance but rather "best" coding practices. A public array can potentially be modified and thus isn't the best approach.

Check out this discussion for more insight: https://stackoverflow.com/questions/9777958/how-to-store-constant-array-of-string-in-java

Reply:
> @vicmanea
> 
> One possibility is using an enum type.
> 
> Another possibility is to change the arrays to private and create getters for the array.
> 
> The issue isn't functionality or performance but rather "best" coding practices. A public array can potentially be modified and thus isn't the best approach.
> 
> Check out this discussion for more insight: https://stackoverflow.com/questions/9777958/how-to-store-constant-array-of-string-in-java

This will be addressed, good advice. Privacy and security is a priority.